### PR TITLE
SaltyBet changed its website title and UI.

### LINKED ISSUE #42: salty.js does not receive any waifu messages after today's saltybet update
> chrome.runtime.onMessage.addListener(function(message, sender, sendResponse) {
>  ...
> }

in salty.js, line 362 is not triggered anymore.

twitch_content.js executes
> chrome.runtime.sendMessage({
> 	message : Waifu4uLines[j]
> }, function(response) {
> 	console.log("response received in twitch content");
> });

(lines 41-45) but nobody receives the messages sent.

This leads to mode, tier, odds and time(?) not being detected anymore.

### PR DESCRIPTION
**Pull request by reconman**
Fixes #42 and detects the illuminati status again.

### CODE REVIEW

File: src/strategy.js [hunk 1]
```diff
@@ -11,22 +11,9 @@ var Strategy = function(sn) {
...
 Strategy.prototype.getBailout = function(tournament){
-	var links = document.getElementsByTagName("a");
-	var target = null;
-	for (var i = 0; i < links.length; i++) {
-		if (links[i].href === 'http://www.saltybet.com/options') {
-			target = links[i];
-		}
-	}
-	var isIlluminati=false;
+	var isIlluminati=document.getElementsByClassName("goldtext").length > 0;
```
Comment (synkarius) [inline src/strategy.js]: This returns true even if you're not logged in. Iluminati names in the left-side bettors box triggers it.

**Comment by synkarius** [comment]
This is a much needed update, but please fix the goldtext thing first.

**Comment by synkarius** [comment]
Ah the heck with it. That SaltyBet update broke a lot of stuff. I'll accept it and fix the goldtext thing myself. Good looking out, @reconman!

**Comment by reconman** [comment]
To my knowledge the goldtext in the left is not important in this context. getBailout is only called when bets are open so there are no bets yet listed in the left window.

**Pull request merged**